src/components/TimeseriesChartMeta/components/TimeseriesMetaInfo.tsx

```diff
@@ -1,4 +1,4 @@
-import Collapse from 'antd/lib/collapse';
+import { Collapse } from 'antd';
```

Comment:
this doesn't change anything for tree shaking right?
just looks nicer?

Reply:
not exactly. In removed line, we imported `Collapse` component from separate file and include code into the bundle. Now we just require it from `antd` library and as `antd` is a peer-dependancy, code will be not included in the bundle and will be required from `antd` library while client app build process.